src/components/common/SubmissionViewer.js

```diff
@@ -1,6 +1,7 @@
-import React, { useState } from 'react';
+import React from 'react';
 import zoomIcon from '../../assets/icons/zoom-icon.svg';
 import lockIcon from '../../assets/icons/lock.svg';
+import { CloseOutlined } from '@ant-design/icons';
```

Comment:
Might want to convert this to svg in the future, but I thought this would suffice for now